pyproject.toml

```diff
@@ -7,22 +7,22 @@ authors = ["Python Discord <info@pythondiscord.com>"]
 [tool.poetry.dependencies]
 python = "3.11.*"
 
-pydis-core = { version = "10.4.0", extras = ["async-rediscache"]}
+pydis-core = { version = "10.7.0", extras = ["async-rediscache"]}
```

Comment:
Didn't bump this to latest as there are breaking changes in v11.0.0 that I don't want to think about in this PR